lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -223,14 +223,16 @@ private void writeDriverHttpEndPoint(final File driverFolder,
     final Path httpEndpointPath = new Path(dfsPath, fileNames.getDriverHttpEndpoint());
 
     String trackingUri = null;
+    LOG.log(Level.INFO, "Attempt to reading " + httpEndpointPath.toString());
```

Comment:
I already have LOG.log(Level.INFO, "Successfully reading trackingUri :" + trackingUri); after successfully read and before break from the loop. 
If after 60 loops, it is still not able to read, this line will be logged 
 LOG.log(Level.WARNING, "Failed reading " + httpEndpointPath.toString());